docs/en/stack/ml/anomaly-detection/ml-limitations.asciidoc

```diff
@@ -262,24 +262,50 @@ that it does not block pop-up windows or create an exception for your {kib} URL.
 //See x-pack-elasticsearch/#844 and x-pack-kibana/#1461
 
 In {kib}, **Anomaly Explorer** and **Single Metric Viewer** charts are not 
-displayed for anomalies that were due to categorization (if model plot is not 
-enabled), `time_of_day` functions, `time_of_week` functions, or `lat_long` 
-geographic functions.
-
-If model plot is not enabled, the charts are not displayed for detectors that 
-use script fields either (except for scripts that define metric fields). In that 
-case, the original source data cannot be easily searched because it has been 
-transformed by the script.
-
-If your <<aggs-dfeeds,{dfeed} uses aggregations with nested `terms` aggs>> and 
-model plot is not enabled for the {anomaly-job}, neither the **Anomaly 
-Explorer** nor the **Single Metric Viewer** can plot and display an anomaly 
-chart for the job. In these cases, the charts are not visible and an explanatory 
-message is shown.
+displayed:
+
+* for anomalies that were due to categorization (if model plot is not 
+enabled),
+* if the datafeed uses scripted fields and model plot is not enabled (except
+for scripts that define metric fields),
+* if your <<aggs-dfeeds,{dfeed} uses aggregations with nested `terms` aggs>> and 
+model plot is not enabled,
+* `freq_rare` functions,
+* `info_content`, `high_info_content`, `low_info_content` functions,
+* `lat_long` geographic functions
+* `time_of_day`, `time_of_week` functions,
+* `varp`, `high_varp`, `low_varp` functions.
+
+Refer to the table below for a more detailed view of supported detector
+functions.
 
 The charts can also look odd in circumstances where there is very little data to 
 plot. For example, if there is only one data point, it is represented as a 
 single dot. If there are only two data points, they are joined by a line.
+The following table shows which detector functions are supported in the Single
+Metric Viewer.
+
+.Detector function support in the Single Metric Viewer
```

Comment:
I think this heading should refer to whether we plot anomaly charts in the Anomaly Explorer and in the Single Metric Viewer